### PR TITLE
Fix ThirdPartyNoticeText.txt encoding

### PR DESCRIPTION
**Pull request by jakebailey**
This is #52800 but using the method I outlined in https://github.com/microsoft/TypeScript/pull/52800#issuecomment-1435787932.

I didn't send this before as I figured the original author might want to do it, but that didn't happen and the PR timed out.

Takes a minute to send, so, here it is.

### CODE REVIEW

File: ThirdPartyNoticeText.txt [hunk 1]
```diff
@@ -162,7 +162,7 @@ For purposes of this definition, the normative portions of the Specification sha
 10.7.8. shall not be considered accepted by an implementer who manifests an intent not to accept the terms of the W3C Community RF Licensing Requirements license as offered by the licensor.
 10.7.9. The RF license conforming to the requirements in this policy shall be made available by the licensor as long as the Specification is in effect. The term of such license shall be for the life of the patents in question.
 I am encouraged to provide a contact from which licensing information can be obtained and other relevant licensing information. Any such information will be made publicly available. 
-10.8. You or Your. �You,� �you,� or �your� means any person or entity who exercises copyright or patent rights granted under this Agreement, and any person that person or entity controls.
+10.8. You or Your. “You,” “you,” or “your” means any person or entity who exercises copyright or patent rights granted under this Agreement, and any person that person or entity controls.
```
Comment (fatcerberus) [inline ThirdPartyNoticeText.txt:165]: I'm not sure this is an improvement... :wink:

Comment (fatcerberus) [reply ThirdPartyNoticeText.txt:165]: Okay, I think GitHub's diff display is just bugged.  It looks fine in my review comment, but under the Files Changed tab all the non-ASCII characters are mangled, whereas the original looks normal.  Does this file need a BOM or something?

Comment (jakebailey) [reply ThirdPartyNoticeText.txt:165]: If that's what the license includes, that's what we include. I'm definitely not going to start arbitrarily replacing characters in someone else's license!

Comment (fatcerberus) [reply ThirdPartyNoticeText.txt:165]: See my second comment.  I wasn't implying you should edit out the smartquotes.

Comment (jakebailey) [reply ThirdPartyNoticeText.txt:165]: I think this file was so broken before that nothing could tell what it was, hence fixing the encoding to UTF-8 without BOM.

Comment (jakebailey) [reply ThirdPartyNoticeText.txt:165]: > See my second comment. I wasn't implying you should edit out the smartquotes.

Hah, yeah, I didn't realize that the other diff view got this wrong.

File: ThirdPartyNoticeText.txt [hunk 2]
```diff
@@ -85,12 +85,12 @@ DOM
 W3C License
 This work is being provided by the copyright holders under the following license.
 By obtaining and/or copying this work, you (the licensee) agree that you have read, understood, and will comply with the following terms and conditions.
-Permission to copy, modify, and distribute this work, with or without modification,�for any purpose and without fee or royalty is hereby granted, provided that you include the following 
+Permission to copy, modify, and distribute this work, with or without modification, for any purpose and without fee or royalty is hereby granted, provided that you include the following
```
Comment (fatcerberus) [inline ThirdPartyNoticeText.txt:88]: image

No idea *what's* going on here.

Comment (jakebailey) [reply ThirdPartyNoticeText.txt:88]: Meanwhile in Outlook 😃 

image

Comment (fatcerberus) [reply ThirdPartyNoticeText.txt:88]: I'm starting to think this file is actually cursed.

Comment (RyanCavanaugh) [reply ThirdPartyNoticeText.txt:88]: There's actually an `0xC2` byte at this position
image

Comment (jakebailey) [reply ThirdPartyNoticeText.txt:88]: And here I thought I had cracked the code.

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by jakebailey** [comment]
Well, it's probably right. I'm going to merge and then we can look at the file afterwards instead, because github sure isn't doing well here.

**Pull request merged**